man/rmd-fragments/home-configuration.Rmd

```diff
@@ -0,0 +1,184 @@
+## Home page
+
+The home page (`index.html`) main content is generated from `pkgdown/index.md`,
+`index.md`, or `README.md`, in that order. Most packages will use `README.md`
+because that's also displayed by GitHub and CRAN. Use `index.md` if you want
+your package website to look different to your README, and use
+`_pkgdown/index.md` if you don't want that file to live in your package
+root directory.
+
+If you use `index.Rmd` or `README.Rmd` it's your responsibility to knit
+the document to create the corresponding `.md`. pkgdown does not do this
+for you because it only touches files in the `doc/` directory.
+
+Extra markdown files in the base directory (e.g. `ROADMAP.md`) 
+or in `.github/` (e.g. `CODE_OF_CONDUCT.md`) are copied by
+`build_home()` to `docs/` and converted to HTML.
+
+The home page also features a sidebar with information extracted from the package.
+You can tweak it via the configuration file, to help make the home page an as
+informative as possible landing page.
+
+## Images and figures
+
+If you want to include images in your `README.md`, they must be stored
+somewhere in the package so that they can be displayed on the CRAN website.
+The best place to put them is `man/figures`. If you are generating figures
+with R Markdown, make sure you set up `fig.path` as followed:
+
+```
+knitr::opts_chunk$set(
+  fig.path = "man/figures/"
+)
+```
+
+This should usually go in a chunk with `include = FALSE`.
+
+````
+```{r chunk-name, include=FALSE}`r ''`
```

Comment:
Is the `` `r '' `` needed?

Reply:
Yes! https://bookdown.org/yihui/rmarkdown-cookbook/verbatim-code-chunks.html

Reply:
But we're not outputting to Rmd here, we're outputting to Rd, and you already have the ````` ```` ````` which pandoc supports (but Rmd does not)

Reply:
Oh, I tried, and the chunk disappeared.

Reply:
Oh bummer; confusing that we have to double escape but no need to look into that now.